Code/CryMP/Client/Client.cpp

```diff
@@ -397,6 +453,10 @@ void Client::ClearKeyBinds()
 {
 	const auto createdInGame = [](KeyBind& bind) -> bool
 	{
+		if (bind.function && bind.createdInGame)
+		{
+			gEnv->pScriptSystem->ReleaseFunc(bind.function);
```

Comment:
`createdInGame` shouldn't have any side effects. It's supposed to only check whether a key bind should be removed or not. I think `SmartScriptFunction` should be used instead of raw `HSCRIPTFUNCTION`, so `ReleaseFunc` is called automatically when needed.